### DISCUSSION TITLE
[@types/node] Missing property readyState on net.Socket

### DISCUSSION THREAD

**Discussion by camo-f** [DefinitelyTyped/DefinitelyTyped#59394] [Issues with a @types package]
Hello,

The property `readyState` seems to be missing on class `Socket` from `net`.
Tested on @types/node v16.11.26

```typescript
import { Socket } from 'net';

const socket: Socket = new Socket();
// The following indicates "Property 'readyState' does not exist on type 'Socket'."
console.log(socket.readyState);
```

Documentation : https://nodejs.org/api/net.html#socketreadystate
Source for Node 16.x : https://github.com/nodejs/node/blob/v16.x/lib/net.js#L555

Thanks

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
fixing: #59397
